core/http/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/statements/StatementsController.java

```diff
@@ -246,13 +248,24 @@ private ModelAndView getSparqlUpdateResult(Repository repository, HttpServletReq
 			}
 		}
 
-		try {
+		final String timeout = request.getParameter(Protocol.TIMEOUT_PARAM_NAME);
```

Comment:
The problem with that is that the protocol module currently has no dependency on javax.servlet, so it can not reference classes like HttpServletRequest. I can of course add the dependency but I'm always a bit hesitant about doing this unless it's necessary. 

However, there is a class ProtocolUtil in the server-spring module, which serves as a sort of go-between. I can probably add it there. I'll have a look.